src/main/kotlin/dev/stalla/builder/AtomBuilder.kt

```diff
@@ -12,25 +12,22 @@ public interface AtomBuilder : Builder<Atom> {
     public fun addAuthorBuilder(authorBuilder: PersonBuilder): AtomBuilder
 
     /** Adds all of the [PersonBuilder] to the list of author builders. */
-    public fun addAllAuthorBuilder(authorBuilders: List<PersonBuilder>): AtomBuilder = apply {
-        authorBuilders.forEach(::addAuthorBuilder)
-    }
+    public fun addAllAuthorBuilder(authorBuilders: List<PersonBuilder>): AtomBuilder =
+        apply { authorBuilders.forEach(::addAuthorBuilder) }
```

Comment:
These are in accordance with the formatting of [expression-bodies](https://kotlinlang.org/docs/coding-conventions.html#expression-bodies) (even though especially scope functions are nice on the same line as the `=`)